robots:
  - robot_1
  - robot_2
  - robot_3

limits:
  x_left: 1
  x_right: 5
  y_bottom: 1
  y_top: 9.5
  height: 10.5
  width: 6

config:
  set_speed: 0.6
  reduced_speed: 0.3
  reduced_buffer: 1
  out_samples: 20
  repulsion: 1
  safe_dist: 1.5
